data/schema.graphql

```diff
@@ -41,11 +41,12 @@ input AddAssetsToConsignmentSubmissionInput {
 
   # A unique identifier for the client performing the mutation.
   clientMutationId: String
+  externalSubmissionId: ID
```

Comment:
You shouldn't update this file manually since it will be updated automatically after Metaphysics PR will be merged.

Reply:
fixed